### PR TITLE
Style usage unification for Astro v1.0

### PR DESCRIPTION
**Pull request by FredKSchott**
- Start Date: 2021-12-14
- Status: Draft

## Summary

Clean up and finalize how CSS is used and referenced in an Astro project.

## Links

- [Full Rendered Proposal](  https://github.com/withastro/rfcs/blob/FredKSchott-patch-1/active-rfcs/0000-style-unification.md)

### CODE REVIEW

File: active-rfcs/0000-style-unification.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+#### 4. ✅ `import './X.css'; // in Astro component frontmatter`
+
+- Supported as a Vite side-effect, but we would not really document this and instead recommend 3. in our documentation and examples.
+- Same build/bundle behavior as if you had done 3.
+- In a previous draft, the proposal was to completely disable this. However, we wouldn't be able to disable you importing a JS file that then did it's own CSS file import, so we decided to just leave the support in for those who ended up using it.
+
+#### 5. ✅ `import './X.css'; // in React component`
+
+- **Usecase:** Supported as the only way to reference a CSS file inside of a React component.
```
Comment (matthewp) [inline active-rfcs/0000-style-unification.md:65]: Nit, this also works in Preact, Solid, Lit, or any 3rd party JavaScript-based component frameworks.

File: active-rfcs/0000-style-unification.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+- Start Date: 2021-12-14
+- Reference Issues: https://github.com/withastro/rfcs/discussions/28
+- Implementation PR: 
+
+# Summary
+
+Clean up and finalize how CSS is used in an Astro project.
```
Comment (matthewp) [inline active-rfcs/0000-style-unification.md:7]: It would be nice if the summary included a quick list of things being added or removed. I think

Comment (FredKSchott) [reply active-rfcs/0000-style-unification.md]: Added another line here to summarize the new method being proposed.

**Comment by matthewp** [comment]
This was approved in the RFC call!

**Pull request merged**